src/app/app.js

```diff
@@ -212,11 +212,33 @@ export class CLI {
 		return false;
 	}
 
+	loadNativeModules(modules) {
+		let errors = [];
+		for (let module of modules) {
+			try {
+				require(module);
+			} catch (err) {
+				errors.push(`Error loading module '${module}': ${err.message}`);
+			}
+		}
+		return errors;
+	}
+
 	run(args) {
 		settings.transitionSparkProfiles();
 		settings.whichProfile();
 		settings.loadOverrides();
 
+		const nativeErrors = this.loadNativeModules(settings.nativeModules);
+		if (nativeErrors.length) {
+			const log = require('./log');
+			for (let error of nativeErrors) {
+				log.error(error);
+			}
+			log.fatal('Please reinstall the CLI again to ensure the module is installed to match the version of node.');
```

Comment:
People with the installer are not likely to see this error. This also works on Windows.